charts/fission-workflows/values.yaml

```diff
@@ -31,3 +31,9 @@ nats:
   cluster: fissionMQTrigger
   location: nats-streaming
   port: 4222
+
+# Enable Debug Message?
+debug: false
+
+# pod deleting grace period
```

Comment:
It probably helps users to add the unit of the period (which is seconds) to this description. 
Something like: `# Pod deleting grace period in seconds`
 `